content/ja/docs/concepts/containers/images.md

```diff
@@ -265,8 +298,9 @@ EOF
    - テナントは、Secretを各NamespaceのimagePullSecretsへ追加する
 
 複数のレジストリへのアクセスが必要な場合、それぞれのレジストリ毎にひとつのSecretを作成する事ができます。
-Kubeletは複数の`imagePullSecrets`を単一の仮想的な`.docker/config.json`にマージします。
 
 ## {{% heading "whatsnext" %}}
 
-* [OCI Image Manifest Specification](https://github.com/opencontainers/image-spec/blob/master/manifest.md)を読みます。
+* [OCI Image Manifest Specification](https://github.com/opencontainers/image-spec/blob/master/manifest.md)を読む
+* [container image garbage collection](/docs/concepts/architecture/garbage-collection/#container-image-garbage-collection)について学ぶ
```

Comment:
```suggestion
* [container image garbage collection](/ja/docs/concepts/architecture/garbage-collection/#container-image-garbage-collection)について学ぶ
```

Reply:
Fixed as your comment and I confirmed the jump on Netlify preview.